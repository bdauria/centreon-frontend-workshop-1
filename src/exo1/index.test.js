import React from "react";
import { render } from "@testing-library/react";
import Exo2 from ".";

describe("Exo2", () => {});