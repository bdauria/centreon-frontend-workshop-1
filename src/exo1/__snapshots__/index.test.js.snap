// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exo1 renders nothing when loading is false 1`] = `null`;

exports[`Exo1 renders with loading indicator when loading is true 1`] = `
<span
  style="font-family: bold; font-size: 14px; color: blue;"
>
  Loading...
</span>
`;
